js/book-carousel.js

```diff
@@ -60,12 +60,14 @@ bcpl.bookCarousel = (($, constants) => {
 			.text('')
 			.append($image);
 
+		$link.addClass('media-title');
```

Comment:
looks like we didn't catch this before, but the classname and selectors are not at the top, will not block for this, since it was already missed